package.json

```diff
@@ -71,8 +71,9 @@
   },
   "typings": "./index.d.ts",
   "dependencies": {
+    "is-buffer": "^1.1.5",
```

Comment:
please remove dependency, we avoided it since it's so small we can just have the function here